cmd/ocm-load-test.go

```diff
@@ -130,15 +138,18 @@ func run(cmd *cobra.Command, args []string) error {
 		time.Duration(viper.GetInt("duration"))*time.Minute,
 		vegetaRate,
 		connection,
-		viper.Sub("tests")); err != nil {
-		return fmt.Errorf("running load test: %v", err)
+		viper.Sub("tests"),
+		logger,
+		cmd.Context()); err != nil {
+		logger.Fatal(cmd.Context(), "running load test: %v", err)
 	}
 
 	return nil
 }
 
 func main() {
-	if err := rootCmd.Execute(); err != nil {
+	ctx := context.Background()
+	if err := rootCmd.ExecuteContext(ctx); err != nil {
 		os.Exit(1)
```

Comment:
Seems like a good candidate for the `log.Fatal("Encountered an unexpected error: ", err)`  (or something along those lines)

Reply:
Yes it would be, but at this point we don't have access to the logger yet :(

Reply:
No problem!